include/boost/math/statistics/univariate_statistics.hpp

```diff
@@ -512,6 +513,58 @@ inline auto interquartile_range(RandomAccessContainer & v)
     return interquartile_range(v.begin(), v.end());
 }
 
+template<class RandomAccessIterator>
+auto mode(RandomAccessIterator first, RandomAccessIterator last)
+{
+    using Real = typename std::iterator_traits<RandomAccessIterator>::value_type;
```

Comment:
Is `Real` the right language here? The probability that a set of real numbers has a mode is zero. I would use `Z`.

Reply:
I might even `static_assert` that `Z` is an integer type.

Reply:
I will make the change to Z. I went with Real for consistency with the rest of the file.